src/components/Layout/Layout.styles.tsx

```diff
@@ -0,0 +1,139 @@
+import styled from 'styled-components';
+
+//components
+import { NavLink } from 'react-router-dom';
+
+//icon
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+//interface
+import { ShowingProps } from '@interface';
+
+//theme
+import theme from '@/theme/theme';
+
+export const LayoutContainer = styled.div`
+    position: sticky;
+    top: 0;
+    width: 100%;
+    z-index: 10;
+`;
+
+export const HeaderWrapper = styled.header``;
+
+export const HeaderImg = styled.div`
+    position: relative;
+    height: 100px;
+    background-image: url('../assets/img/header.png');
+    background-size: cover;
+    background-position: 0 10%;
+
+    @media screen and (${theme.device.tablet}) {
+        height: 110px;
+    }
+
+    @media screen and (${theme.device.laptop}) {
+        height: 120px;
+    }
+`;
+
+export const HeaderText = styled.div`
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    text-align: center;
+    z-index: 1;
+
+    h1 {
+        font-size: 1.6em;
+        text-transform: uppercase;
+        letter-spacing: 2px;
+
+        @media screen and (${theme.device.mobileL}) {
+            margin-bottom: 2px;
+            white-space: nowrap;
+        }
+
+        @media screen and (${theme.device.laptop}) {
+            font-size: 2em;
+        }
+    }
+
+    p {
+        display: none;
+
+        @media screen and (${theme.device.mobileL}) {
+            display: inline-block;
+            margin-top: 2px;
+        }
+    }
+`;
+
+export const HeaderBackground = styled.div`
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background-color: ${theme.color.primaryLight};
+`;
+
+export const NavigationWrapper = styled.nav`
+    @media screen and (${theme.device.mobileL}) {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        background-color: ${theme.color.primary};
+        height: 3rem;
+    }
+`;
+
+export const NavigationBurger = styled.button<ShowingProps>`
+    position: absolute;
+    padding: 5px;
+    transform: translate(0, -100%);
```

Comment:
Transform jest chyba niepotrzbne, tak samo jak padding w pixelach.

Reply:
musi być transform, żeby ten burger był na headerze. padding w px też - bo to ikonka burgera, która jest stała